Example/Stagehand/InteractiveAnimationViewController.swift

```diff
@@ -0,0 +1,181 @@
+//
+//  Copyright 2025 Block Inc.
```

Comment:
nice of you

Reply:
Not sure how this is supposed to work in a fork 😅